idalib/src/idb.rs

```diff
@@ -213,6 +214,36 @@ impl IDB {
         }
     }
 
+    pub fn get_cmt(&self, ea: Address, rptble: bool) -> String {
+        unsafe { idalib_get_cmt(ea.into(), rptble) }
+    }
+
+    pub fn set_cmt(
+        &self,
+        ea: Address,
+        comm: impl AsRef<str>,
+        rptble: bool,
+    ) -> Result<(), NulError> {
+        let s = CString::new(comm.as_ref())?;
+        unsafe {
+            set_cmt(ea.into(), s.as_ptr(), rptble);
+        }
+        Ok(())
+    }
+
+    pub fn append_cmt(
```

Comment:
I'd change this to be `append_cmt_with` and have a second variant, so we can have a default API that gives us non-repeatable comment:

```
pub fn append_cmt(&self, ea: Address, comm: impl AsRef<str>) -> Result<(), ..> {
    self.append_cmt_with(ea, comm, false)
}
```